recipes/cmor/meta.yaml.in

```diff
@@ -0,0 +1,40 @@
+package:
+    name: cmor
+    version: @VERSION@
+
+source:
+    git_rev: @UVCDAT_BRANCH@
+    git_url: git://github.com/PCMDI/cmor.git
```

Comment:
I also wonder about the `git://` vs `https://` which is what is noted in the documentation https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-from-git

@dnadeau4 @doutriaux1

Reply:
Conda does not mind it and build works.